Bottom: f497a3e81656ed56c1c5e4352fe294a2e97b856f
Top:    f19bd133a6f47a57ac64a3d93ef7bbe3e194ffa2
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:03:31 +0000

Remaining (bug?) from another patch..

NB: Original function had this bug before we moved it


---

diff --git a/src/misc.c b/src/misc.c
index 16f69a9..f8096ec 100644
--- a/src/misc.c
+++ b/src/misc.c
@@ -397,7 +397,7 @@ calc_arc_from_points_and_included_angle (PointType *p1, PointType *p2, Angle inc
   hx = p1->X + (p2->X - p1->X) / 2;
   hy = p1->Y + (p2->Y - p1->Y) / 2;
 
-  p_to_h_dist = hypot (p2->X - p1->X, p2->Y - p1->Y) / 2.;
+  p_to_h_dist = hypot (p2->X - p1->Y, p2->Y - p1->Y) / 2.;
   c_to_h_dist = p_to_h_dist / tan (TO_RADIANS (included_angle) / 2.);
 
   unit_hcx = (double)-(hy - p1->Y) / p_to_h_dist;
